forge/routes/api/project.js

```diff
@@ -356,8 +357,24 @@ module.exports = async function (app) {
      * @name /api/v1/project/:id
      * @memberof forge.routes.api.project
      */
-    app.put('/:projectId', { preHandler: app.needsPermission('project:edit') }, async (request, reply) => {
+    app.put('/:projectId', { preHandler: app.needsPermission('project:member') }, async (request, reply) => {
```

Comment:
The `preHandler` function at the top of this file already does the work to ensure the requesting user is a member of the team.

```suggestion
    app.put('/:projectId', async (request, reply) => {
```